src/net_processing.cpp

```diff
@@ -1754,7 +1757,10 @@ void static ProcessGetData(CNode& pfrom, const CChainParams& chainparams, CConnm
 {
     AssertLockNotHeld(cs_main);
 
-    std::deque<CInv>::iterator it = pfrom.vRecvGetData.begin();
+    PeerRef peer = GetPeerRef(pfrom.GetId());
+    if (peer == nullptr) return;
```

Comment:
2d9f2fca43aadcdda4d644cddab36dca88b40b97, nit:
```suggestion
    if (!peer) return;
```